scripts/convert_environment.py

```diff
@@ -121,19 +121,21 @@ def env() -> Mapping[str, Optional[str]]:
 
 
 def convert_value(value: str) -> Optional[str]:
+    """
+    Convert shell-style interpolations to Python str.format() templates.
+    Translate `$foo` and `${foo}` to `{foo}`.
+    Translate special case '{{lambda_name}}' to '*'.
```

Comment:
… but I'm pretty sure it should not implement this special case. The case should not occur. The string `{lambda_name}` should be gone from the code base.